docs/user/discover.asciidoc

```diff
@@ -275,7 +270,24 @@ image:images/discover-maps.png[Map containing documents]
 [[share-your-findings]]
 === Share your findings
 
-To share your findings with a larger audience, click *Share* in the *Discover* toolbar. For detailed information about the sharing options, refer to <<reporting-getting-started,Reporting>>.
+To share your findings with a larger audience, click *Share* in the *Discover* toolbar.
+For detailed information about the sharing options, refer to <<reporting-getting-started,Reporting>>.
+
+[float]
+[[alert-from-Discover]]
+=== Generate an alert
+
+From *Discover*, you can create an alert to periodically
+check when data goes above or below a certain threshold within a given time interval.
+
+. Ensure that your data view,
+query, and filters fetch the data for which you want an alert.
+. In the toolbar, click *Alerts > Create search threshold rule*.
++
+The *Create rule* form is pre-filled with the latest query sent to {es}.
+. <<rule-type-es-query, Configure your query>> and <<action-types, select a connector type>>.
+
+. Click *Save*.
```

Comment:
Not mandatory, but since this is just a single type of rule offered here, it might be nice to refer them to more information about all the other types of rules they can create in the other apps.  For example, something like:
```suggestion
. Click *Save*.

For more about this and other rules provided in {kib} {alert-features}, go to <<alerting-getting-started>>.
```